lib/MetaCPAN/API/Model/Cover.pm

```diff
@@ -0,0 +1,33 @@
+package MetaCPAN::API::Model::Cover;
+
+use MetaCPAN::Moose;
+
+with 'MetaCPAN::API::Model::Role::ES';
+
+sub find_release_coverage {
+    my ( $self, $release ) = @_;
+
+    my $query = +{ term => { release => $release } };
+
+    my $res = $self->_run_query(
+        index => 'cover',
+        type  => 'cover',
+        body  => {
+            query => $query,
+            size  => 999,
+        }
+    );
+    use DDP;
+    p $res;
```

Comment:
Leftover debugging?

Reply:
Oops, trying to do too many things at one time. Cleaning it out in an additional commit that I’ll squash after approval before it gets merged.